.travis.yml

```
@@ -7,10 +7,9 @@ os:
   - osx
 
 node_js:
-  - 6
   - 8
   - 10
-  - node
```

Generelly we should test 8, 10, 12 and node. Because we want to fail early =)